src/backend/call-service-in-container

```diff
@@ -173,6 +178,7 @@ else
 fi
 
 CONTAINER_VOLUMES="-v $OUTEROUTDIR:$INNEROUTDIR -v $OUTERSRCDIR:$INNERSRCDIR -v $OUTERHOMEDIR:$INNERHOMEDIR -v $MOUNTDIR$INNERSCRIPTDIR:$INNERSCRIPTDIR:ro"
+[ -n "$USE_SERVICE_CONFIG" ] && CONTAINER_VOLUMES="$CONTAINER_VOLUMES -v /etc/obs/services/$USE_SERVICE_CONFIG:/etc/obs/services/$USE_SERVICE_CONFIG:ro"
```

Comment:
Reminder to myself:

<nitpicking>
if we need to mount "/etc/obs/services" in future, we easily can change

`USE_SERVICE_CONFIG="scm-bridge"` to `USE_SERVICE_CONFIG="/scm-bridge"`

and

`-v /etc/obs/services/$USE_SERVICE_CONFIG:/etc/obs/services/$USE_SERVICE_CONFIG:ro`

to

`-v /etc/obs/services$USE_SERVICE_CONFIG:/etc/obs/services$USE_SERVICE_CONFIG:ro`

and a service which needs _ALL_ directories in  `/etc/obs/services` simply sets

`USE_SERVICE_CONFIG="/"`